$OpenBSD: patch-etc-share,v 1.2 2000/09/18 06:56:53 camield Exp $

Usage of /usr/local/etc is deprecated on OpenBSD. Use /usr/local/share
instead.

--- dnscache-conf.c.orig	Mon Sep 18 00:36:57 2000
+++ dnscache-conf.c	Mon Sep 18 08:45:21 2000
@@ -88,10 +88,10 @@
   if (chdir(auto_home) == -1)
     strerr_die4sys(111,FATAL,"unable to switch to ",auto_home,": ");
   seed_addtime();
-  fdrootservers = open_read("etc/dnscache/@");
+  fdrootservers = open_read("share/dnscache/@");
   seed_addtime();
   if (fdrootservers == -1)
-    strerr_die4sys(111,FATAL,"unable to open ",auto_home,"/etc/dnscache/@: ");
+    strerr_die4sys(111,FATAL,"unable to open ",auto_home,"/share/dnscache/@: ");
   seed_addtime();
 
   init(dir,FATAL);
